### ISSUE TITLE
Annotation end does not line up with sequence

### ISSUE THREAD

**Issue by haasleah** [Lattice-Automation/seqviz#45]
**Describe the bug**
When rendering seqviz with a longer sequence and annotation, the end of the annotation and sequence do not line up.

**To Reproduce**
- Replicable with the following part:
```txt
annotations: Array(1)
    0:
        color: "#80D849" 
        direction: "REVERSE" 
        end: 266
         name: "Promoter" 
        start: 0 
        type: "PROMOTER"
name: "pC^rbs" 
seq: "tgccgatacaagaacaacagcgcgttgagcgcctgccggtgggtggccggcgccacttgcttctcggtggcgagcatggtcagaaaaccctcgacttcagcttgccccatttcgcgcggatgtcgaaacccaccaaggctgcgggccgtccacaacacaaatgccttggcccagtagacataagccttctcggtctgtaggctgtaatgcaggtagcgaacccgttcaagaaataattttgtttaactttaagaaggagatatacat"
```

**Expected behavior**
The end of the sequence and the end of the annotation should line up assuming the endIndex is the same length as the sequence - 1.

**Screenshots**
<img width="1302" alt="Screen Shot 2019-12-04 at 2 12 34 PM" src="https://user-images.githubusercontent.com/45974914/70173107-23d59600-16a0-11ea-80ef-a1c48a295664.png">

**Desktop (please complete the following information):**

- OS: iOS
- Browser: Chrome
- Version: 78

**Comment by JJTimmons** [comment]
Related but different issue, adding to this one:

![Screen Shot 2019-12-05 at 1 27 02 PM](https://user-images.githubusercontent.com/13923102/70262784-f69dec00-1762-11ea-996a-95318ac476d6.png)

**Comment by JJTimmons** [comment]
Hey @haasleah, let me know whether you continue to see this. My changes resolved the issue I was having in my screen-shot. Am hoping they were caused by the same thing.

**Comment by haasleah** [comment]
Hmm, I updated to version 0.5.3 and I'm still seeing the issue on my side. Though it looks like the last production build was around noon today. Did the changes you're talking about make it in?

**Comment by JJTimmons** [comment]
Ah that's my bad. Just pushed and merged 0.5.5.

More changes soon for the bpColors issue

**Comment by haasleah** [comment]
Yes, this looks great. Thanks for fixing it so quickly! I'll let you know if we see any other related issues.
